### ISSUE TITLE
Has the security group membership limit been lifted?

### ISSUE THREAD

**Issue by mgoto-git** [MicrosoftDocs/powerapps-docs#2477]
[Enter feedback here]

Before , there was a limitation of security group membership that must be under 1,000 .
Today , the mention has changed to "Your Power Apps administrator may have applied the limit to the maximum size of the security group. By default, this limit is 10,000 members."

It is unclear if we can change default limit to larger or only to smaller and  details how to change the default size 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4dcdc292-8c6a-240c-cb87-375fed2bc7e7
* Version Independent ID: 203e06b4-a126-9b8f-56ce-6afdf790d633
* Content: [Set permission and share your app using Power Apps in Teams - Power Apps](https://docs.microsoft.com/en-us/powerapps/teams/set-perms-and-share#assign-the-colleagues-with-access-role-to-a-security-group-and-share-the-app)
* Content Source: [powerapps-docs/teams/set-perms-and-share.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/teams/set-perms-and-share.md)
* Service: **powerapps**
* GitHub Login: @matthewbolanos
* Microsoft Alias: **mabolan**

**Comment by jathorpe** [comment]
Any update on the clarification of the limit?

**Comment by mgoto-git** [comment]
I've got a suggestion from  MS Support below.
I hope this would be updated in the document.

----------------------------------------------------------------------------------
・The limitation update has updated to 10,000 .
 ( This is default setting . It cant be changed to larger)

・How to confirm

1. Confirm current setting
Title : Get-TenantSettings
URL : https://docs.microsoft.com/en-us/powershell/module/microsoft.powerapps.administration.powershell/get-tenantsettings?view=pa-ps-latest

2. Update setting
Title : Set-TenantSettings
URL : https://docs.microsoft.com/en-us/powershell/module/microsoft.powerapps.administration.powershell/set-tenantsettings?view=pa-ps-latest

For example , ( both need Add-PowerAppsAccount and sign in first )

1.
$setting = Get-TenantSettings
$setting.powerPlatform.teamsIntegration
← We got 10,000 from this response

2.
$requestBody = @{
powerPlatform = @{
teamsIntegration = @{
shareWithColleaguesUserLimit=50000}}}

Set-TenantSettings -RequestBody $requestBody
← We got update to 50,000
